pkg/koordlet/metricsadvisor/utils.go

```diff
@@ -0,0 +1,19 @@
+package metricsadvisor
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+// NOTE: functions in this file can be overwritten for extension
+
+// getRuntimeClassName returns RuntimeClassName which refers to a RuntimeClass object in
+// the node.k8s.io group, which should be used to run this pod.
+func getRuntimeClassName(pod *corev1.Pod) *string {
+	return pod.Spec.RuntimeClassName
+}
+
+// shouldIgnorePod will return true if Pod uses RuntimeClass other than runc.
+func shouldIgnorePod(pod *corev1.Pod) bool {
+	runtimeClass := getRuntimeClassName(pod)
```

Comment:
return true directly?

Reply:
Even though this function might be overwritten, I think we should still keep this simple check. WDYT?